### PR TITLE
provide detailed alt text to numpy image 01 - Zero Index .

### PR DESCRIPTION
**Pull request by blankdots**
Zero Index now has a detailed content: ../fig/python-zero-index.png

Related to https://github.com/swcarpentry/python-novice-inflammation/issues/781

Note: Followed the recommendations at: https://webaim.org/techniques/alttext/#basics trying to be as specific as possible for the content and the function.

- [x] `02-numpy.md > Zero Index`

### CODE REVIEW

File: _episodes/02-numpy.md [hunk 1]
```diff
@@ -242,7 +242,10 @@ It takes a bit of getting used to,
 but one way to remember the rule is that
 the index is how many steps we have to take from the start to get the item we want.
 
-![Zero Index](../fig/python-zero-index.png)
+![data array variable illustrating the 0 based index. For a data array variable equal to 
+data = \[\['A','B','C'\],\['D','E','F'\],\['G','H','I'\]\] data\[0\]\[0\]='A', data\[0\]\[1\]='B', 
+data\[0\]\[2\]='C', data\[1\]\[0\]='D', data\[1\]\[1\]='E', data\[1\]\[2\]='F', 
+data\[2\]\[0\]='G', data\[2\]\[1\]='H', data\[2\]\[2\]='I'](../fig/python-zero-index.png)
```
Comment (ldko) [inline _episodes/02-numpy.md]: It makes sense to me that you have translated the text shown in the image to the `alt` text. I wonder how well the text conveys the meaning when being read by a screenreader.

Comment (blankdots) [reply _episodes/02-numpy.md]: I am at a loss here as well :( 
One thing is to test it or use some browser accessibility addons to see how this gets parsed.

**Comment by blankdots** [comment]
closing in favour of #922

**Comment by ldko** [comment]
Thanks @blankdots. Your work on this is appreciated!

**Pull request closed without merging**